src/twisted/test/test_sslverify.py

```diff
@@ -1049,7 +1049,7 @@ def test_tlsProtocolsNoMethodWithAtLeast(self):
                 privateKey=self.sKey,
                 certificate=self.sCert,
                 method=SSL.SSLv23_METHOD,
-                raiseMinimumTo=sslverify.TLSVersion.TLSv1_2,
+                raiseMinimumTo=sslverify.TLSVersion.TLSv1_3,
```

Comment:
All these tests which change the flags but don't change any assertions probably should not be changing; their behavior should remain the same.